src/main/java/ee/buerokratt/ruuter/domain/steps/http/HttpQueryArgs.java

```diff
@@ -12,4 +14,16 @@ public class HttpQueryArgs {
     private HashMap<String, Object> query;
     private HashMap<String, String> headers;
     private HashMap<String, Object> body;
+
+    public void addHeaders(ConfigurationInstance ci) {
```

Comment:
Something that annoys me a bit, is that the whole ci object is added here. The method would make more sense, if simply accepted a list of Headers that it adds, but currently its executing "business logc" - this isn't really the place to do it. The business logic should be moved to the steps, where that data is needed.